amp/TEMPLATE/reampv2/src/modules/geocoder/components/layout/table/ActivityTable.jsx

```diff
@@ -1,35 +1,40 @@
-import React, {Component} from 'react';
+import React, {Component, useState} from 'react';
 import BootstrapTable from "react-bootstrap-table-next";
 import paginationFactory from 'react-bootstrap-table2-paginator';
 import {bindActionCreators} from "redux";
 import {connect} from "react-redux";
-import TableActions from "./ActivityActions";
 
 import './table.css';
-import GeocodedActivity from "./LocationsTabel";
+import {
+    loadActivities,
+    selectActivityForGeocoding,
+    selectAllActivitiesForGeocoding
+} from "../../../actions/activitiesAction";
+import {Loading} from "../../../../../utils/components/Loading";
+
+const SelectedActivitiesMessage = (props) =>  {
+    return (
+        <>
+           <div className="activities-message"><b>{props.size}</b> activities selected for geocoding</div>
+        </>
+    );
+}
 
 class ActivityTable extends Component {
     constructor(props) {
         super(props);
-        this.state = { selectedRowAction: null };
 
-        this.handleActionsClick = this.handleActionsClick.bind(this);
+        this.handleOnSelect = this.props.onSelectActivity;
```

Comment:
if we are not binding this to the method received by props there is no point on creating a variable we can just take it from props. This way the code is easier toread